src/lib/hasExample.ts

```
@@ -1,4 +1,15 @@
+import { useTheme } from '../composables/useTheme'
+import { formatJson } from './formatJson'
+
 export function hasExample(schema: any, visited: Set<any> = new Set(), level: number = 0): boolean {
```

Biome

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)